timemachine/cpp/src/neighborlist.cu

```diff
@@ -153,23 +167,26 @@ void Neighborlist<RealType>::build_nblist_device(
     const double *d_coords,
     const double *d_box,
     const double cutoff,
+    int *d_rebuild_nblist,
     cudaStream_t stream) {
 
     // assert(N == N_);
 
-    // reset the interaction count
-    gpuErrchk(cudaMemsetAsync(d_ixn_count_, 0, 1*sizeof(*d_ixn_count_), stream));
+    // Only need a single thread/block to reset the ixn count based on whether or not to rebuild
+    k_reset_interaction_count<<<1, 1, 0, stream>>>(d_ixn_count_, d_rebuild_nblist);
```

Comment:
we can probably fuse this into another kernel somewhere, but I'm fine with this being here for now.